processors/acoustic_extractor.py

```diff
@@ -211,6 +212,11 @@ def __extract_utt_acoustic_features(dataset_output, cfg, utt):
             label = audio_to_label(wav, cfg.preprocess.bits)
             save_feature(dataset_output, cfg.preprocess.label_dir, uid, label)
 
+        if cfg.preprocess.extract_acoustic_token:
```

Comment:
Do not modify `__extract_utt_acoustic_features` anymore. It is not a common extraction pipeline now. See `extract_utt_acoustic_features_tts` (line221) and `extract_utt_acoustic_features_vocoder`(line233) as reference. Please move all the functions of TTS's acoustic feature extraction into line221.